basin-helper

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PATH_TO_JSON=/tmp/basin.json
+PATH_TO_FILES=$1
+PATH_TO_SHARD=$2
```

Comment:
Not entirely sure I understand this. I don't think we need to enforce any path based on the app_id, the shards we create doesn't necessarily need to belong to an specific app, and we can try these shards with any app using the custom-content option I am adding to the e-k-l.